content/scalers/aws-sqs.md

```diff
@@ -13,48 +13,78 @@ Scale applications based on AWS SQS Queue.
 
 ### Trigger Specification
 
-This specification describes the `aws-sqs-queue` trigger for AWS SQS Queue.
+This specification describes the `aws-sqs-queue` trigger that scales based on an AWS SQS Queue.
 
 ```yaml
 triggers:
   - type: aws-sqs-queue
+    authenticationRef: 
+      name: keda-trigger-auth-aws-role
     metadata:
       # Required: queueURL
-      queueURL: https://sqs.eu-west-1.amazonaws.com/<acccount_id>/testQueue
-      # Optional: region
-      awsRegion: "eu-west-1"
-      # Optional: AWS Access Key ID
```

Comment:
Since they are optional I would rather keep them for now similar to https://keda.sh/scalers/aws-cloudwatch/ until we have clarification that it's backward compatible or not.

Opened https://github.com/kedacore/keda/issues/453 to verify this.